snap/snapcraft.yaml

```diff
@@ -340,6 +340,7 @@ apps:
     daemon: simple
```

Comment:
Need to add the -cp or --configProvider option to specify using the configuration provider.  --registry is now just service registration. with no value specified it defaults to consul.http://localhost:8500.

 I recommend adding just  `--configProvider`